cmd/modules/analyzers/test-analyzer/main.go

```diff
@@ -85,9 +86,11 @@ func (testanalyzer *TestAnalyzer) Analyze(controlService service.ControlServiceC
 		if test == "TestPackageNode" {
 			testfunction = TestPackageNode
 		} else if test == "TestCalcBuildGraph" {
-			testfunction = TestCalcBuildGraph
+			expectedTargets = []string{"Calculator/calc", "Calculator/libcalc.so"}
```

Comment:
We really need to get rid of those magic values and use the module configuration for this.